### PR TITLE
insertionSort

### PR DESCRIPTION
**Pull request by Trainee0**

### CODE REVIEW

File: Sort/InsertitonSort.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package Sort;
+
+class InsertionSort {
```
Comment (ShitCodeTester) [inline Sort/InsertitonSort.java]: class should be public

File: Sort/InsertitonSort.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            System.out.print(arr[i] + " ");
+        }
+    }
+
+    public static void main(String[] args) {
+        int[] arrNotSorted = new int[10];
+        for (int i = 0; i < arrNotSorted.length; i++) {
+            arrNotSorted[i] = (int) (Math.random() * 100);
+        }
+        InsertionSort ins = new InsertionSort();
```
Comment (ShitCodeTester) [inline Sort/InsertitonSort.java]: print initial array before sorting

**Review by ShitCodeTester — CHANGES_REQUESTED** [review]
After those changes code could be merged

**Comment by Trainee0** [comment]
done

**Pull request merged**